src/main/java/com/gooddata/visualizationObject/VisualizationType.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+
+package com.gooddata.visualizationObject;
+
+public enum VisualizationType {
+    TABLE,
+    LINE,
+    COLUMN,
+    BAR,
+    PIE,
+
+    // fallback to prevent 500
+    UNKNOWN;
+
+    public String toString() {
+        return name().toLowerCase();
+    }
+
+    public static VisualizationType get(final String type) {
+        for (VisualizationType vizType : VisualizationType.values()) {
```

Comment:
how about to use `valueOf(toUpperCase)`? Also pls adjust the method name (get is not very descriptive)